src/DynamicLibrary.cpp

```diff
@@ -0,0 +1,59 @@
+#include "DynamicLibrary.hpp"
+
+#include <string>
+
+#ifdef WIN32
+#include <windows.h>
+
+class WindowsDynamicLibrary
+	: public mia::DynamicLibrary
+{
+	HMODULE handle = nullptr;
+	std::string dllName;
+public:
+	virtual ~WindowsDynamicLibrary()
+	{
+		unload();
+	}
+
+	virtual void load(const char* name) override
```

Comment:
Maybe return bool to indicate whether it succeeded or not. Information about what coused loading error may also be helpfull.

Reply:
Done.